vmdb/db/migrate/20150503094838_add_container_group_ip_to_container_groups.rb

```diff
@@ -0,0 +1,9 @@
+class AddContainerGroupIpToContainerGroups < ActiveRecord::Migration
+  def up
+    add_column :container_groups, :ip_address, :string
```

Comment:
Can you please rename to ipaddress (no underscore) for consistency?  I should have made the "no underscore" part clearer.